src/main/java/io/confluent/kafkarest/v2/KafkaConsumerManager.java

```diff
@@ -493,15 +481,12 @@ private KafkaConsumerState getConsumerInstance(String group, String instance) {
     return getConsumerInstance(group, instance, false);
   }
 
+  /*
+    Update the expiration of a KafkaConsumerState.
+    Synchronized to avoid race conditions with the ExpirationThread
+   */
   private synchronized void updateExpiration(KafkaConsumerState state) {
```

Comment:
This method is contributing much anymore. Perhaps we can remove it?

Reply:
Yeah I thought about that initially. I wrongly remembered that the `updateExpiration` is stateful by using the `expirationTime` instance variable but it was only setting it by using config variables.